data/attributes.yml

```diff
@@ -30209,6 +30209,46 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3413
+  name: Comic edition
+  description: Indicates the specific release version or edition of a comic book, influencing its rarity and collectible value within the market
+  friendly_id: comic_edition
+  handle: comic-edition
+  values:
+  - comic_edition__anniversary_issue
+  - comic_edition__collector's_edition
+  - comic_edition__digital_edition
+  - comic_edition__director's_cut
+  - comic_edition__first_edition
+  - comic_edition__international_edition
+  - comic_edition__reprint
+  - comic_edition__signed_edition
+  - comic_edition__variant_cover
+- id: 3414
+  name: Comic tradition
+  description: The Comic tradition embodies the unique style and themes in comic books
```

Comment:
This description looks slightly off when compared to other attributes. Suggestion:
"Indicates the cultural or regional tradition from which the comic originates, such as Chinese comics or manga"